static/js/components/GroupAdmissionRequestsManagement.jsx

```diff
@@ -0,0 +1,187 @@
+    return <></>;
+  }
+  const requests = groupAdmissionRequests[groupID];
+
+  const handleAcceptRequest = async ({ requestID, userID }) => {
+    const addGroupUserResult = await dispatch(
+      groupsActions.addGroupUser({ userID, admin: false, group_id: groupID })
+    );
+    const updateAdmissionRequestStatusResult = await dispatch(
+      groupAdmissionRequestsActions.updateAdmissionRequestStatus({
+        requestID,
+        status: "accepted",
+      })
+    );
+    if (
+      addGroupUserResult.status === "success" &&
+      updateAdmissionRequestStatusResult.status === "success"
+    ) {
+      dispatch(showNotification("Successfully admitted user to group."));
+      dispatch(groupsActions.fetchGroups(true));
+      dispatch(groupActions.fetchGroup(groupID));
+      dispatch(
+        groupAdmissionRequestsActions.fetchGroupAdmissionRequests(groupID)
+      );
+    }
+  };
+
+  const handleDeclineRequest = async ({ requestID }) => {
+    const updateAdmissionRequestStatusResult = await dispatch(
+      groupAdmissionRequestsActions.updateAdmissionRequestStatus({
+        requestID,
+        status: "declined",
+      })
+    );
+    if (updateAdmissionRequestStatusResult.status === "success") {
+      dispatch(showNotification("Successfully declined request."));
+      dispatch(
+        groupAdmissionRequestsActions.fetchGroupAdmissionRequests(groupID)
+      );
+    }
+  };
+
+  const renderActions = (dataIndex) => {
+    const request = requests[dataIndex];
+    if (request.status === "pending") {
+      return (
+        <>
+          <Button
+            size="small"
+            variant="contained"
+            onClick={() =>
+              handleAcceptRequest({
+                requestID: request.id,
+                userID: request.user_id,
+              })
+            }
+            data-testid={`acceptRequestButton${request.user_id}`}
+          >
+            Accept
+          </Button>
+          <Button
+            size="small"
+            variant="contained"
+            onClick={() => handleDeclineRequest({ requestID: request.id })}
+            data-testid={`declineRequestButton${request.user_id}`}
+          >
+            Decline
+          </Button>
+        </>
+      );
+    }
+    if (request.status === "declined") {
+      return (
+        <Button
+          size="small"
+          variant="contained"
+          onClick={() =>
+            handleAcceptRequest({
+              requestID: request.id,
+              userID: request.user_id,
+            })
+          }
+          data-testid={`acceptRequestButton${request.user_id}`}
+        >
+          Accept
+        </Button>
+      );
+    }
+    return <></>;
+  };
+
+  const renderUserInfo = (dataIndex) => {
+    const { user } = requests[dataIndex];
+    let userInfoString = user.username;
+    if (!!user.first_name && !!user.last_name) {
+      userInfoString += ` (${user.first_name} ${user.last_name})`;
+    }
+    return userInfoString;
+  };
```

Comment:
```suggestion
  const renderUserInfo = (value) => {
    let userInfoString = value.username;
    if (!!value.first_name && !!value.last_name) {
      userInfoString += ` (${value.first_name} ${value.last_name})`;
    }
    return userInfoString;
  };
```
And set `customBodyRenderLite: renderUserInfo` below to `customBodyRender: renderUserInfo` - that will allow using the search tool to search the usernames and user names